lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -23,6 +19,7 @@ class FeatureCard extends React.Component {
     this.campaigns = [];
     this.campaignsToExclude = setting('dsUser.activity.signups', []);
     this.campaignsApiPage = 1;
+    this.apiClient = new ApiClient(window.location.hostname, '/api/v1/', window.location.port);
```

Comment:
Ya. Was also debating a bit, whether we let the dev specify full thing `/api/v1/` or just a `v1` and we fill out the rest in the constructor when building the url? I see benefits with either... not something that can't be easily changed anyhoo.